meeting-notes/CMWG-20231121.md

```diff
@@ -0,0 +1,86 @@
+# UK TRE Community management Working Group - 21/11/23
+
+## Links
+
+* [Note of notes](https://hackmd.io/AFbOeD2XQ6KyQvzZrIj6VQ)
```

Comment:
What permissions are on the notes linked from this link? Can anyone with the links edit those notes?

Reply:
Whoops good catch yeah! I'll change permission to signed in users - mb we remove this reference from the repo?

Reply:
If this is an archive of notes I think it needs to be read-only for everyone, or not linked from the public notes (probably easiest, if you do this can you force push the old commit away?)

Reply:
```suggestion

```